legend-engine-xt-persistence-grammar/src/main/java/org/finos/legend/engine/language/pure/dsl/persistence/compiler/toPureGraph/PersistenceCompilerExtension.java

```diff
@@ -90,6 +100,43 @@ public Iterable<? extends Processor<?>> getExtraProcessors()
                             purePersistenceContext._platform(HelperPersistenceContextBuilder.buildPersistencePlatform(persistenceContext.platform, context));
```

Comment:
side comment - given we support extensions here, we might want to move this to later compilation phase, in case they reference other pakageable elements.  This might also apply to trigger.

Reply:
Ack'd -- makes sense to bump a step later for breathing room. Will do in the next PR